README.md

```diff
@@ -1,4 +1,49 @@
-# README
+# Street Art Gallery - Backend Service
 
-## Implementation Instructions 
-(Tailor to someone using rails for the first time) 
+## Introduction
+
+This application is a Ruby on Rails API service, utilizing GraphQL to create user-friendly endpoints. Our database is built using Postgres. You can find our schema in the section below.
+
+## Schema
+
+## Endpoints
```

Comment:
These are placeholders that will get filled in later once everything is complete.